app/views/idv/doc_auth/_start_over_or_cancel.html.erb

```diff
@@ -11,5 +11,7 @@ locals:
           class: 'usa-button usa-button--unstyled',
         ) { t('doc_auth.buttons.start_over') } %>
   <% end %>
-  <%= render 'shared/cancel', link: idv_cancel_path(step: local_assigns[:step]) %>
+  <div class="margin-top-2 padding-top-1 border-top border-primary-light">
```

Comment:
the cancel link has its own `border-top` right? How come this doesn't create a duplicate border?

I'd expect we'd pass in a class to override the one in the _cancel link already?

Reply:
With these changes, we're no longer using the `shared/cancel` partial, so we won't get the additional border from there. We'll only have the one between "Start Over" and "Cancel". 

I had considered if it might be better to continue reusing that partial and be able to customize it, but:

1. We weren't really using much of its built-in logic anyways, since we explicitly set the link
2. As implemented here is now more aligned with the [React component of the same name](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture/components/start-over-or-cancel.jsx)
3. With upcoming changes in LG-5258 to Start Over/Cancel experience, this is probably going to have to be redone anyways

Reply:
Ah I must have read the diff backwards... moving *away* from the shared partial, this makes more sense now